### PR TITLE
DROCC Code

### PR DESCRIPTION
**Pull request by SachinG007**
drocc_trainer : /pytorch/edgeml_pytorch/trainer
Example Files : /examples/pytorch/DROCC/
1) Tabular datasets 
2) Timeseries datasets
3) Cifar10 data

Please try to check if you are able to follow the readme.

### CODE REVIEW

File: examples/pytorch/DROCC/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Deep Robust One-Class Classification 
+In this directory we present examples of how to use the `DROCCTrainer` to replicate results in [paper](https://proceedings.icml.cc/book/4293.pdf).
+
```
[the remark this answers is not in the document]
Comment (SachinG007) [reply examples/pytorch/DROCC/README.md:3]: done

File: examples/pytorch/DROCC/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+python3  main_timeseries.py --hd 128 --lr 0.00001 --lamda 0.5 --gamma 2 --ascent_step_size 0.1 --radius 10 --batch_size 256 --epochs 200  --optim 0 --restore 0 --metric AUC -d "root_data"
+```
+
+## CIFAR Experiments
+```
+python3  main_cifar.py  --lamda 1  --radius 8 --lr 0.001 --gamma 1 --ascent_step_size 0.001 --batch_size 256 --epochs 40 --optim 0 --normal_class 0
+```
+
+
+## Arguments Detail
```
[the remark this answers is not in the document]
Comment (SachinG007) [reply examples/pytorch/DROCC/README.md]: done

File: examples/pytorch/DROCC/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+#### Thyroid
+```
+python3 main_tabular.py --hd 128 --lr 0.001 --lamda 1 --gamma 2 --ascent_step_size 0.001 --radius 2.5 --batch_size 256 --epochs 100 --optim 0 --restore 0 --metric F1 -d "root_data"
+```
+
+#### Abalone 
+```
+python3 main_tabular.py --hd 128 --lr 0.001 --lamda 1 --gamma 2 --ascent_step_size 0.001 --radius 3 --batch_size 256 --epochs 200 --optim 0 --restore 0 --metric F1 -d "root_data"
```
[the remark this answers is not in the document]
Comment (SachinG007) [reply examples/pytorch/DROCC/README.md:51]: added annotation on number of train samples and test samples

**Comment by SachinG007** [comment]
@harsha-simhadri  addressed the comments. Please approve the PR
